packages/grid/x-data-grid-pro/src/tests/rowEditing.DataGridPro.test.tsx

```diff
@@ -168,7 +168,7 @@ describe('<DataGridPro /> - Row Editing', () => {
           apiRef.current.setEditCellValue({ id: 0, field: 'currencyPair', value: ' usdgbp ' }),
         );
         await act(() => apiRef.current.setEditCellValue({ id: 0, field: 'price1M', value: 100 }));
-        expect(renderEditCell1.lastCall.args[0].row).to.deep.equal({
+        expect(renderEditCell2.lastCall.args[0].row).to.deep.equal({
```

Comment:
No idea why this was working in the first place. Reading the test case, AFAICT, this is the correct check.